docs/section5/Rule5-29_nonprimary.md

```diff
@@ -0,0 +1,38 @@
+# Envelope - Rule 5-29  
+**Schema Version** 0.0.23  
+**Primary Rule:** True
+**Rule ID:** 5-29  
+**Rule Description:** Baseline fenestration shall be assumed to be flush with the exterior wall, and no shading projections shall be modeled.
+**Appendix G Section:** Section G3.1-5(d) Building Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:**  None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+  1. get_opaque_surface_type()
+  2. get_surface_conditioning_category()
+
+## Rule Logic:
+
+- Get surface conditioning category dictionary for B_RMR: ```scc_dictionary_b = get_surface_conditioning_category(B_RMR)```
+
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```
+
+  - For each zone in building segment: ```for zone_b in building_segment_b.zones:```
+
+    - For each surface in zone: ```for surface_b in zone_b.surfaces:```
```

Comment:
For brevity, we usually skip the path to the target object in the RDSs if it's self evident. So this RDS could have started with "for each surface ...". @claperle can you please confirm?

Reply:
@mkarpman I believe the way I have written it follows the same structure as the other rules in this section. All "for" loops that I am seeing in the RDS are written starting from the building_segment level. However, when the rules are written in python I know the path skips directly to the target objects.